conda_env/specs/__init__.py

```diff
@@ -16,6 +22,12 @@
 
 
 def detect(**kwargs):
+    # Check file existence if --file was provided
```

Comment:
This part is probably better located elsewhere, maybe as a separate function, just to better isolate concerns? changing it here made it simpler, otherwise we would need to change in different cli methods like create and update to add a file check before the spec detection

Thoughts?